include/swift/IDE/RefactoringKinds.def

```diff
@@ -46,6 +46,7 @@ RANGE_REFACTORING(ExtractFunction, "Extract Method", extract.function)
 
 RANGE_REFACTORING(ExtractRepeatedExpr, "Extract Repeated Expression", extract.expr.repeated)
 
+RANGE_REFACTORING(ConvertStringsConcatenationToInterpolation, "Convert to Strings Interpolation", convert.string-concatenation.interpolation)
```

Comment:
Nit: This should be `Convert to String Interpolation`

Reply:
Thanks, fixed!